.changeset/young-countries-confess.md

```diff
@@ -0,0 +1,6 @@
+---
+"@primer/react": patch
```

Comment:
Just noting that its marked as `minor` on the description

Reply:
Ah, yes you're right! Updated